posts/JHipster-Conf-2-Hottest-JHipster-Day-of-the-Year.md

```diff
@@ -0,0 +1,59 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- JHipster
+date: 2019-07-01T17:27:07.000Z
+title: "JHipster Conf 2 - Hottest JHipster Day of the Year"
+image: https://github.com/falydoor/blog-usa/blob/jhipster-conf-2/images/2019/07/jhipster-conf.png
+---
+
+June 27 was one of the hottest days (90°F) in beautiful Paris, but it was also the hottest JHipster day of the year with its worldwide event: the JHipster Conf 2! 300 people attended (almost twice last year's number) from all over the world with 19 different countries including Australia and South Africa. With English and French tracks, the conference covered many different topics with speakers from the JHipster core team and from big organizations such as Google or Pivotal.
+
+This year was also different since we were happy to welcome speakers with real-life stories which make the beauty of the Open Source community even bigger. [Pia Mancini](https://twitter.com/piamancini) who is the cofounder of [Open Collective](https://opencollective.com/) gave an awesome talk on how Open Collective helps Open Source communities by being financially transparent. [Cindy Marin](https://twitter.com/laslorma) and [Ernesto Butto](https://twitter.com/poolebu) shared their really touching story on how JHipster started a positive chain of events, you can read more about it on [Medium](https://medium.com/@ernesto_31246/a-thank-you-letter-to-the-jhipster-team-7397ad8a09fb).
```

Comment:
```suggestion
This year was also different since we were happy to welcome speakers with real-life stories which make the beauty of the Open Source community even better. [Pia Mancini](https://twitter.com/piamancini), who is the cofounder of [Open Collective](https://opencollective.com/), gave an awesome talk on how Open Collective helps Open Source communities by being financially transparent. [Cindy Marin](https://twitter.com/laslorma) and [Ernesto Butto](https://twitter.com/poolebu) shared their really touching story on how JHipster started a positive chain of events, you can read more about it on [Medium](https://medium.com/@ernesto_31246/a-thank-you-letter-to-the-jhipster-team-7397ad8a09fb).
```